### PR TITLE
Change the way strings are concated more efficiently

### PR DESCRIPTION
**Pull request by marload**
It is not recommended to write `+=` when concating string in Python.

Thank You!

### CODE REVIEW

**Comment by askhade** [comment]
CIs are failing because of flake8 error:
./onnx/tools/net_drawer.py:61:1: W293 blank line contains whitespace

**Comment by marload** [comment]
Complete!

File: onnx/tools/net_drawer.py [hunk 1]
```diff
@@ -50,14 +50,15 @@ def _form_and_sanitize_docstring(s):  # type: (Text) -> Text
...
-        for i, output in enumerate(op.output):
-            node_name += '\n output' + str(i) + ' ' + output
-        node = pydot.Node(node_name, **kwargs)
+            node_name = [f'{op.op_type} (op#{op_id})']
+
+        for i, inp in enumerate(op.input):
+            node_name.append(f'input{i} {inp}')
+        for i, outp in enumerate(op.output):
+            node_name.append(f'output{i} {outp}')
+        node = pydot.Node('\n '.join(node_name), **kwargs)
```
Comment (askhade) [inline onnx/tools/net_drawer.py:61]: why are you adding a line break here?

**Comment by askhade** [comment]
@marload : there is 1 open comment apart from that the PR looks good.

**Pull request closed without merging**